tools/depends/target/dav1d/Makefile

```diff
@@ -0,0 +1,52 @@
+include ../../Makefile.include
+include DAV1D-VERSION
+DEPS = ../../Makefile.include Makefile
+
+# configuration settings
+CONFIGURE = $(NATIVEPREFIX)/bin/python3 $(NATIVEPREFIX)/bin/meson \
+	--buildtype=release \
+	--prefix=$(PREFIX) \
+	--libdir=lib \
+	--default-library=static \
+	-Denable_asm=true \
+	-Denable_tools=false \
+	-Denable_examples=false \
+	-Denable_tests=false
+
+ifeq ($(CROSS_COMPILING), yes)
+CONFIGURE += --cross-file $(PREFIX)/share/cross-file.meson
+export CC=$(CC_FOR_BUILD)
+export CXX=$(CXX_FOR_BUILD)
+export CFLAGS=$(CFLAGS_FOR_BUILD)
+export CXXFLAGS=$(CXXFLAGS_FOR_BUILD)
+else
+export CC CXX CFLAGS CXXFLAGS
+endif
+export PKG_CONFIG_LIBDIR=$(PREFIX)/lib/pkgconfig
```

Comment:
bit of a weird reference for now, but according to https://github.com/mesonbuild/meson/blob/master/docs/markdown/Cross-compilation.md

the key quote states

> Since 0.54.0 The pkg_config_libdir property may point to a list of path used internally by Meson to set the PKG_CONFIG_LIBDIR environment variable for pkg-config. This prevents pkg-config from searching cross dependencies in system directories.

the kicker, there is no 0.54.0 tagged release right now, so more a possible todo in the future, to move this over to the cross file as well.